concurrency-limits-servlet/src/test/java/com/netflix/concurrency/limits/ConcurrencyLimitServletFilterTest.java

```diff
@@ -75,12 +78,48 @@ public void testDoFilterThrottledCustomStatus() throws ServletException, IOExcep
         ConcurrencyLimitServletFilter filter = new ConcurrencyLimitServletFilter(limiter, customThrottleStatus);
 
         //Empty means to throttle this request
-        when(limiter.acquire(any())).thenReturn(Optional.empty());
+        doReturn(Optional.empty()).when(limiter).acquire(any());
 
         MockHttpServletResponse response = new MockHttpServletResponse();
 
         filter.doFilter(new MockHttpServletRequest(), response,  new MockFilterChain());
 
         assertEquals(customThrottleStatus, response.getStatus(), "custom status should be respected");
     }
+
+    @Test
+    public void testDoFilterBypassCheckPassed() throws ServletException, IOException {
+
+        ConcurrencyLimitServletFilter filter = new ConcurrencyLimitServletFilter(limiter);
+
+        MockHttpServletRequest request = new MockHttpServletRequest("GET", "/admin/health");
+        MockHttpServletResponse response = new MockHttpServletResponse();
+        MockFilterChain filterChain = new MockFilterChain();
+
+        filter.doFilter(request, response, filterChain);
+
+        assertEquals(request, filterChain.getRequest(), "Request should be passed to the downstream chain");
+        assertEquals(response, filterChain.getResponse(), "Response should be passed to the downstream chain");
+
+        // Acquire a bypass listener
```

Comment:
We should verify that we got a bypass listener here.

Reply:
Updated tests with metrics validation